README.md

```diff
@@ -1,48 +1,56 @@
 # Getting-started-with-.NET-MAUI-ProgressBar-
+The [.NET MAUI ProgressBar](https://www.syncfusion.com/maui-controls/maui-progressbar?utm_source=github&utm_medium=listing&utm_campaign=maui-progressbar-github-samples) is a control that shows task progress with customizable visuals. A quick-start project that helps you create and configure the Syncfusion .NET MAUI ProgressBar control. This project also contains code snippet to configure the control’s basic features that explained in the introduction blog. 
 
-A quick-start project that helps you create and configure the Syncfusion .NET MAUI ProgressBar control. This project also contains code snippet to configure the control’s basic features that explained in the introduction blog. 
+## Project pre-requisites
 
-## Supported platforms
+Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/) with .NET MAUI workloads and [.NET SDK 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later version in your machine before starting to work on this project.
 
-.NET Multi-platform App UI (.NET MAUI) apps can be written for the following platforms:
+## How to run this application?
 
-* Android 5.0 (API 21) or higher.
-* iOS 11 or higher, using the latest release of Xcode.
-* macOS 10.15 or higher, using Mac Catalyst.
-* Windows 11 and Windows 10 version 1809 or higher, using [Windows UI Library (WinUI) 3](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/).
```

Comment:
Why this section rempved, and please do not add you own content here just interlink the top section -features etc